### PR TITLE
Styling changes, editing error messages, displaying menu categories

### PR DESCRIPTION
**Pull request by konradscheng**
https://app.asana.com/0/1173499440842807/1190889100108599/f

### CODE REVIEW

File: src/narrow-screen/screens/RestaurantScreen.js [hunk 1]
```diff
@@ -112,14 +131,14 @@ export default class extends React.Component {
...
-          : 
+          :
           (
-            this.state.error ? 
-            <div>Some error has ocurred. Please try reloading the page.</div> :
-            <div>Loading...</div>
+            this.state.error ?
+            <PageError>There was an error loading this page. Please try reloading the page or contact the Nomi team by filling out a form at dinewithnomi.com</PageError> :
+            <RestaurantNotFound>Restaurant Not Found - Please navigate to a different restaurant.</RestaurantNotFound>
           )
```
Comment (Tony1023) [inline src/narrow-screen/screens/RestaurantScreen.js:139]: This part should not be an error yet. The page is actually really LOADING.

**Review by Tony1023 — CHANGES_REQUESTED** [review]
I think the files on this branch are very out of date. Could you merge in the most recent changes on develop?

**Comment by Tony1023** [comment]
For example, the states on the wide-screen/screens/RestaurantScreen and narrow-screen/screens/RestaurantScreen are elevated to screens/RestaurantScreen as of https://github.com/Nomi-Technologies/web-menu/pull/19

File: src/narrow-screen/screens/RestaurantScreen.js (hunk 1 above)
Comment (konradscheng) [reply src/narrow-screen/screens/RestaurantScreen.js]: changed it back, also I think files should be up to date now?

File: src/components/SharedDishTile.js [hunk 2]
```diff
@@ -54,7 +68,7 @@ export default function (props) {
         <TitleTrailing>{props.dish.price}</TitleTrailing>
       </TileTitle>
       {
-        props.dish.description ? 
+        props.dish.description ?
         (
           <TileSubtitle>
             <Description>{props.dish.description}</Description>
```
Comment (Tony1023) [inline src/components/SharedDishTile.js:74]: Font of description needs to be sans sarif as well?

File: src/narrow-screen/components/FilterSlideUpPanel.js [hunk 3]
```diff
@@ -138,7 +138,7 @@ const SectionTitle = styled.i`
 `;
 
 const SaveButton = styled(Button)`
-  background-color: #F3A35C;
+  background-color: #F9D5B4 !important;
```
Comment (Tony1023) [inline src/narrow-screen/components/FilterSlideUpPanel.js:141]: idk why the asana task says this hex value, but it's apparently not the same as what's in figma... Can we double check on that?
Screen Shot 2020-09-02 at 5 01 20 PM

File: src/narrow-screen/components/MenuCategoryPanel.js [hunk 4]
```diff
@@ -8,9 +8,24 @@ const DishList = styled.div`
...
+  flex: none;
+  order: 0;
+  align-self: center;
+  margin: 10px 0px;
+`;
+
 export default function(props) {
   return (
     <DishList>
+      <CategoryTitle>{props.category}</CategoryTitle>
```
Comment (Tony1023) [inline src/narrow-screen/components/MenuCategoryPanel.js:28]: I don't think this is working...?

**Review by Tony1023 — COMMENTED** [review]
Let me push a fix that removes the redundant error check in narrow-screen/screens/MenuScreen.js

File: src/narrow-screen/components/FilterSlideUpPanel.js (hunk 3 above)
Comment (konradscheng) [reply src/narrow-screen/components/FilterSlideUpPanel.js:141]: hmm this was something @cchyung told me to change, should I change it back?

File: src/narrow-screen/components/MenuCategoryPanel.js (hunk 4 above)
Comment (konradscheng) [reply src/narrow-screen/components/MenuCategoryPanel.js]: should be fixed, think the merge messed something up

**Review by Tony1023 — COMMENTED** [review]
I think we need to install Source Serif Pro font but perhaps not in this PR. Let's wait for @cchyung to confirm the color of the apply filter button and we're good to merge!

**Review by Tony1023 — APPROVED** [review]

File: src/narrow-screen/components/FilterSlideUpPanel.js (hunk 3 above)
Comment (cchyung) [reply src/narrow-screen/components/FilterSlideUpPanel.js:141]: zach and i went over this personally - we should use the hex code from asana

**Pull request merged**